packages/@ember/-internals/runtime/lib/system/object_proxy.ts

```diff
@@ -1,4 +1,4 @@
-import FrameworkObject from './object';
+import { FrameworkObject } from '@ember/object';
```

Comment:
This is actually a code change since previous this was really `EmberObject` not `FrameworkObject. However, I believe it is what was intended.